packages/http-client-java/generator/http-client-generator-test/src/test/java/azure/resourcemanager/resources/ArmResourceTest.java

```diff
@@ -44,11 +46,417 @@ public class ArmResourceTest {
     private static final String LOCATION_RESOURCE_ID
         = "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource";
     private static final String LOCATION_RESOURCE_TYPE = "Azure.ResourceManager.Resources/locationResources";
+    private static final String EXTENSION_RESOURCE_SUBSCRIPTION_SCOPE_URI
+        = "/subscriptions/00000000-0000-0000-0000-000000000000";
+    private static final String EXTENSION_RESOURCE_RESOURCE_GROUP_SCOPE_URI
+        = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg";
+    private static final String EXTENSION_RESOURCE_RESOURCE_SCOPE_URI
+        = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Resources/topLevelTrackedResources/top";
+    private static final String EXTENSION_RESOURCE_TENANT_SCOPE_URI = "";
+    private static final String EXTENSION_RESOURCE_NAME = "extension";
+    private static final String EXTENSION_RESOURCE_TYPE = "Azure.ResourceManager.Resources/extensionsResources";
+    private static final String EXTENSION_RESOURCE_BASE_ID
+        = "/providers/Azure.ResourceManager.Resources/extensionsResources/extension";
     private final ResourcesManager manager
         = ResourcesManager.authenticate(ArmUtils.createTestHttpPipeline(), ArmUtils.getAzureProfile());
 
     @Test
-    public void tetsLocationResources() {
+    public void testExtensionResources() {
```

Comment:
Can you separate this into e.g. 4 tests, on different scopes?

It is too long here.

Reply:
Fixed by the [commit#4252949](https://github.com/microsoft/typespec/pull/7597/commits/4252949d861d1d4bea9b641b2bd3ed90b3adf248)